mkt/websites/models.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from django.conf import settings
+from django.db import models
+
+import json_field
+
+from mkt.site.models import ModelBase
+from mkt.translations.fields import TranslatedField
+from mkt.tags.models import Tag
+from mkt.translations.fields import save_signal
+
+
+class Website(ModelBase):
+    default_locale = models.CharField(max_length=10,
+                                      default=settings.LANGUAGE_CODE)
+    url = TranslatedField()
+    title = TranslatedField()
+    short_title = TranslatedField()
+    description = TranslatedField()
+    keywords = models.ManyToManyField(Tag)
+    # FIXME regions
+    submitter_description = TranslatedField()
+    categories = json_field.JSONField(default=None)
+    icon_type = models.CharField(max_length=25, blank=True)
```

Comment:
We'll need a `fetch_icon` task similar to the one that exists for webapps, but since the one for webapps parses manifest and what not, it'll have to be different. We should reuse the image resizing stuff though, and copy `get_icon_url()`